### PR TITLE
Move runService to services-shared

### PR DESCRIPTION
**Pull request by znewton**
Tinylicious depends on `runService` from services-utils; however, #5663 moved it to server-routerlicious (unexposed, unconsumable by Tinylicious). Then, #5801 moved it to server-routerlicious-base (exposed, many dependencies). In #6053, it became clear that we cannot add routerlicious-base as a dependency of Tinylicious just to get `runService`.

As discussed with @GaryWilber, this PR moves `runService` to services-shared which explicitly exists to "[implement] service interfaces that are shared between Routerlicious and Tinylicious." We believe that this should be the final, as-permanent-as-can-be spot for `runService`.

### CODE REVIEW

**Review by tanviraumi — APPROVED** [review]
:shipit:

**Review by GaryWilber — APPROVED** [review]

File: server/routerlicious/packages/services-shared/src/runner.ts [hunk 1]
```diff
@@ -5,53 +5,7 @@
 
 import { inspect } from "util";
 import nconf from "nconf";
-import { ILogger } from "@fluidframework/server-services-core";
-
-/**
```
Comment (hedasilv) [inline server/routerlicious/packages/services-shared/src/runner.ts:10]: Were these also moved to `server-services-core`, or are they already there? Sorry if I missed something

Comment (hedasilv) [reply server/routerlicious/packages/services-shared/src/runner.ts:10]: My comment didn't highlight the lines I intended; I meant whether `IRunner`, `IResources`, `IResourcesFactory ` and `IRunnerFactory ` are also in `server-services-core`

Comment (znewton) [reply server/routerlicious/packages/services-shared/src/runner.ts:10]: They're already there, they were duplicate. Not sure if they were leftover from one of the other moves around Runner interfaces, or it was a copy-paste error, but yeah, these are just duplicates from services-core now

Comment (hedasilv) [reply server/routerlicious/packages/services-shared/src/runner.ts:10]: Got it! Thanks for taking care of this!

**Comment by anthony-murphy** [comment]
have you considered moving tinylicious into the serve packages to avoid issues like this?

**Pull request merged**